Scripts/script-ContextGetPathForString.yml

```diff
@@ -0,0 +1,46 @@
+commonfields:
+  id: ContextGetPathForString
+  version: -1
+name: ContextGetPathForString
+script: |+
+  var path = args.path
+  var str = args.str;
+  var flat = {};
+
+  function searchString(obj) {
+      flattenFields(obj,undefined,flat);
+      keysArr = Object.keys(flat);
+      for (i = 0; i < keysArr.length - 1; i++) {
```

Comment:
why `keysArr.length - 1` and not `keysArr.length`?
It will not look for the last key. right?